httpie/client.py

```diff
@@ -128,28 +130,42 @@ def get_requests_kwargs(args, base_headers=None):
     Translate our `args` into `requests.request` keyword arguments.
 
     """
+    write_info('get_requests_kwargs','Total: 9')
     # Serialize JSON data, if needed.
     data = args.data
     auto_json = data and not args.form
+    if isinstance(data, dict):
+        has_branched('get_request_kwargs', 1)
+        if args.json:
+            has_branched('get_requests_kwargs', 2)
+        if auto_json:
```

Comment:
`elif`?

Reply:
what if both ``args.json`` and ``auto_json`` are true? Should we not write both branch ID's then since it's a ``or`` in the "real" ``if``?

Reply:
But if the or is lazy and the first part is true, the second one will never get checked?